### DISCUSSION TITLE
Function _winapi.SetNamedPipeHandleState does not exist

### DISCUSSION THREAD

**Discussion by webbanza** [IronLanguages/ironpython3#1754] [Q&A] [answered]
Original CPython:
https://github.com/python/cpython/blob/main/Modules/_winapi.c#L1658

Ironpython3 Usage:
https://github.com/IronLanguages/ironpython3/blob/master/Src/StdLib/Lib/multiprocessing/connection.py#L556

Code Snippet:
test.py
```
from multiprocessing import Process, Pipe

def f(conn):
    conn.send([42, None, 'hello'])
    conn.close()


parent_conn, child_conn = Pipe()
p = Process(target=f, args=(child_conn,))
p.start()
a = parent_conn.recv()   # prints "[42, None, 'hello']"
print(a)
p.join()
```

Run:
`ipy test.py`

Error:
```
Traceback (most recent call last):
  File "C:\code\test.py", line 34, in <module>
  File "C:\Program Files\IronPython 3.4\lib\multiprocessing\context.py", line 61, in Pipe
  File "C:\Program Files\IronPython 3.4\lib\multiprocessing\connection.py", line 556, in Pipe
AttributeError: 'module' object has no attribute 'SetNamedPipeHandleState'
```

**Comment by webbanza** [comment]
![ss1](https://github.com/IronLanguages/ironpython3/assets/6710614/d69de285-9f7a-46c4-87ad-42d82cd2bd51)

@BCSharp @slozier Kindly help.

**Comment by slozier** [comment, marked as answer]
The `multiprocessing` module still needs a lot of work. I don't mind adding `_winapi.SetNamedPipeHandleState` to unblock, but I think you will just hit another failure further down the line. For example on the very next line `overlapped = _winapi.ConnectNamedPipe(h1, overlapped=True)` will raise a `NotImplementedError`.
